gcp/api/README.md

```diff
@@ -3,18 +3,23 @@
 The API server runs on Cloud Run with Cloud Endpoints.
 
 # API server updates (backend)
+
 If the backend code (i.e. server.py) code is changed, follow these steps:
 
-Pick a TAG (base this on the date. e.g. 20200101).
+Pick a TAG to use for the build (base this on the date. e.g. date +"%Y%m%d")).
+
+Note: if you may need to run `sudo gcloud auth gcloud auth configure-docker` the
```

Comment:
Actually, this command looks wrong. This needs to be 2 separate commands right?

Reply:
Oh wow, something went wrong with the paste, fixed.